triplestore/src/triplestore/utils.py

```diff
@@ -0,0 +1,128 @@
+from __future__ import annotations
+

```

Comment:
It seems that you didn't do `git mv network_utils.py utils.py`, so the Git history will lose all details about the previous file and record that the new one spontaneously appears now.

Just FYI.